modules/smartxBidAdapter.js

```diff
@@ -0,0 +1,400 @@
+           * TODO: what about the pre_market_bid partners sizes?
+           */
+          utils._each(currentBidRequest.params.pre_market_bids, function (pmb) {
+            if (pmb.deal_id == smartxBid.id) {
+              smartxBid.price = pmb.price;
+              serverResponseBody.cur = pmb.currency;
+            }
+          });
+          const bid = {
+            requestId: currentBidRequest.bidId,
+            currency: serverResponseBody.cur || 'USD',
+            cpm: smartxBid.price,
+            creativeId: smartxBid.crid || '',
+            ttl: 360,
+            netRevenue: true,
+            vastContent: smartxBid.adm,
+            vastXml: smartxBid.adm,
+            mediaType: VIDEO,
+            width: smartxBid.w,
+            height: smartxBid.h
+          };
+          const context = utils.deepAccess(currentBidRequest, 'mediaTypes.video.context');
+          if (context === 'outstream') {
+            const playersize = utils.deepAccess(currentBidRequest, 'mediaTypes.video.playerSize');
+            const renderer = Renderer.install({
+              id: 0,
+              url: '//',
+              config: {
+                adText: 'SmartX Outstream Video Ad via Prebid.js',
+                player_width: playersize[0][0],
+                player_height: playersize[0][1],
+                content_page_url: utils.deepAccess(bidderRequest, 'data.site.page'),
+                ad_mute: +!!utils.deepAccess(currentBidRequest, 'params.ad_mute'),
+                hide_skin: +!!utils.deepAccess(currentBidRequest, 'params.hide_skin'),
+                outstream_options: utils.deepAccess(currentBidRequest, 'params.outstream_options'),
+                outstream_function: utils.deepAccess(currentBidRequest, 'params.outstream_function')
+              }
+            });
+            try {
+              renderer.setRender(outstreamRender);
+              renderer.setEventHandlers({
+                impression: function impression() {
+                  return utils.logMessage('SmartX outstream video impression event');
+                },
+                loaded: function loaded() {
+                  return utils.logMessage('SmartX outstream video loaded event');
+                },
+                ended: function ended() {
+                  utils.logMessage('SmartX outstream renderer video event');
+                }
+              });
+            } catch (err) {
+              utils.logWarn('Prebid Error calling setRender or setEventHandlers on renderer', err);
+            }
+            bid.renderer = renderer;
+          }
+          bidResponses.push(bid);
+        })
+      });
+    }
+    return bidResponses;
+  }
+}
+
+function createOutstreamScript(bid) {
+  // const slot = utils.getBidIdParameter('slot', bid.renderer.config.outstream_options);
+  utils.logMessage('[SMARTX][renderer] Handle SmartX outstream renderer');
+  const elementId = bid.adUnitCode;
+  // eslint-disable-next-line camelcase
```

Comment:
Is it possible that lines 335-340 are causing the problem? As far as i can remember it was pointed out in the course of the PR that we should define the callback functions as empty variables before to pass the lint test.